02-express-tutorial/01-http-basics.js

```diff
@@ -0,0 +1,27 @@
+const http = require('http');
+
+const server = http.createServer((req, res) => {
+    //console.log(req.method)
+
+    const url = req.url;
+    //home page
+    if (url === '/'){
+        res.writeHead(200, {'content-type': 'text/html'})
+        res.write('<h1>home page</h1>')
+        res.end()
+    }
+    //about page
+    else if(url === '/about'){
```

Comment:
Challenge question: do you remember why we use `if` / `if else` / `else` rather than just a handful of `else` statements here?